Adds database backup template (#127)
airplanedev/templates: database_backup/README.md

# Template: Database backup

## Next steps

- Navigate to the database_backup directory: `cd database_backup`

> - You'll need to get a access key and secret from your AWS Account. Once you have done this, store them in the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` config variables. You should also modify the `AWS_REGION` and `AWS_BUCKET` environment variables to store your default region and bucket name. 

Reviewer: ```suggestion
- You'll need to get a access key and secret from your AWS Account. Once you have done this, store them in the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` config variables. You should also modify the `AWS_REGION` and `AWS_BUCKET` environment variables to specify your AWS region and S3 bucket name. 
```

> - You'll need to get a access key and secret from your AWS Account. Once you have done this, store them in the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` config variables. You should also modify the `AWS_REGION` and `AWS_BUCKET` environment variables to store your default region and bucket name. 

Reviewer: ```suggestion
- Get an access key and secret from your AWS account. Once you have done this, store them in the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` config variables. You should also modify the `AWS_REGION` and `AWS_BUCKET` environment variables to store your default region and bucket name. 
```

> - Also add the config variable to your dev config file in order to [develop tasks locally](https://docs.airplane.dev/dev-lifecycle/dev-config-file#add-a-config-variable).

Reviewer: ```suggestion
- [Define these config variables in Studio](https://docs.airplane.dev/dev-lifecycle/dev-config-file#add-a-config-variable) in order to develop this task locally.
```

- If you plan on using a [database resource](https://docs.airplane.dev/resources/overview) other than the demo database, change the `demo_db` resource in the `database_backup.task.yaml` file to the slug of your own database resource.
- Develop your template locally: `airplane dev`
- Deploy your tasks and view: `airplane deploy --yes`

> - Visit the docs to learn more about how to build views: https://docs.airplane.dev/views/getting-started

Reviewer: This mentions Views not tasks

Author: Updated
